### PR TITLE
Update intermediate HTML CSS/ CSS positioning lesson/Additional resources section: Adding a video

### PR DESCRIPTION
**Pull request by Finn245**
### **Because:**
This PR enhances the CSS positioning lesson by integrating insights and examples from the 'Slay the Dragon' video tutorial, which recaps the CSS positioning concepts covered in the lesson with such  an easy examples.

### **This PR:**
- Incorporates the video tutorial on CSS positioning from 'Slay the Dragon' to enrich the lesson content.
- Enhances the lesson with visual demonstrations for improved clarity.

### **Additional Information:**
This update enriches the learning experience by providing supplemental material that reinforces key concepts covered in the CSS positioning lesson.

### CODE REVIEW

File: README.md
Comment (MaoShizhong) [inline README.md:1]: Could you also undo the changes to the README to remove it from this PR? It's unrelated and would be best done in a separate PR. Many thanks

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
Thanks @Finn245 

Could you resolve the merge conflict please? While doing this, could you also extend the link text for your added video to include"CSS positioning from"? So the link text should read "CSS positioning from Slaying the Dragon". That would make it sufficiently descriptive and accessible.

File: README.md
Comment (Finn245) [reply README.md:1]: Hello, it is done.

**Comment by MaoShizhong** [comment]
Looks like there's still the merge conflict to resolve.
I see you tried to resolve it by fetching and merging `origin/main`, your remote fork, except your remote fork is out of sync with upstream main (TOP's repo).

You can either sync your remote fork with upstream on GH, then redo the fetch and merge resolution to origin main, or you can skip the first step by just fetching from `upstream main` and merging `upstream/main` instead (i.e. syncing your local fork directly with TOP's repo).

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 1]
```diff
@@ -69,3 +69,7 @@ The following questions are an opportunity to reflect on key topics in this less
 This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
 - [Understand the CSS Position Property With Practical Examples](https://www.makeuseof.com/css-position-property-practical-examples/) provides some different CSS methods for positioning elements.
+ 
+- You can check out this helpful video resource on CSS positioning from [CSS positioning from Slaying the Dragon](https://www.youtube.com/watch?v=MxEtxo_AaZ4&t=2s) for clear explanations and practical examples.
```
Comment (MaoShizhong) [inline intermediate_html_css/intermediate_css_concepts/positioning.md:73]: Just meant to move the opening `[` forward to extend the link text, no need to double up on the words here
```suggestion
- You can check out this helpful video resource on [CSS positioning from Slaying the Dragon](https://www.youtube.com/watch?v=MxEtxo_AaZ4&t=2s) for clear explanations and practical examples.
```

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
Almost there! Just need the below quick change as well as removing the trailing space on line 72, and the extra blank lines at the end of the file.
Our lesson files need to end with only a single blank line, but it looks like there are 3 blank lines instead.

**Comment by Finn245** [comment]
Oh okay so i have to remove line 72 and remove the last line which is the 75 so we keep only 1 blank line(74 one)?

**Comment by MaoShizhong** [comment]
> Oh okay so i have to remove line 72 and remove the last line which is the 75 so we keep only 1 blank line(74 one)?

Yep. Either remove the space on line 72 (it's not blank, it's a line with just a space in it), or remove that line altogether (we don't need it anyway). Then files must end with only a single blank line, so delete any extra blank lines at the end of the file.

**Review by MaoShizhong — APPROVED** [review]
Many thanks for the new resource, it's a nice video on the topic for sure :+1:

**Comment by Finn245** [comment]
Thank you for your kind words! I'm thrilled that you found the new resource helpful. I'm looking forward to contributing more and continuing to enhance this project . Your encouragement means a lot, and I'm excited to collaborate further 😊👍.

**Pull request merged**